static/app/views/onboarding/integrationSetup.tsx

```diff
@@ -187,8 +173,55 @@ function IntegrationSetup(props: Props) {
     );
   };
 
+  if (isLoading) {
+    return <LoadingIndicator />;
+  }
+
+  if (!docsConfig || !dsn || !projectKeyId) {
+    return (
+      <LoadingError
+        message={t(
+          'The getting started documentation for this platform is currently unavailable.'
+        )}
+        onRetry={refetch}
+      />
+    );
+  }
```

Comment:
In the future, we might consider moving this to the parent component, as we currently have the same check within `SdkDocumentation`